2009-05-16  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/ia64/sysconf.c (__sysconf): use
	_POSIX_VERSION instead of 200112L.

--- a/sysdeps/unix/sysv/linux/ia64/sysconf.c
+++ b/sysdeps/unix/sysv/linux/ia64/sysconf.c
@@ -33,7 +33,7 @@ long int
 __sysconf (int name)
 {
   if (name == _SC_CPUTIME || name == _SC_THREAD_CPUTIME)
-    return has_cpuclock () ? 200112L : -1;
+    return has_cpuclock () ? _POSIX_VERSION : -1;
 
   /* Everything else is handled by the more general code.  */
   return linux_sysconf (name);
